src/login-screen.js

```diff
@@ -1,8 +1,44 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
 
-let LoginScreen = () =>
-    <div>
-        <h1>Login here!</h1>
-    </div>
+class LoginScreen extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            username: '',
+            password: '',
+        }
+    }
 
+    render() {
+        return <div>
+            <h1>Please Login</h1>
+            <form 
+            onSubmit={event => {
+                event.preventDefault();
+                this.props.dispatch({
+                    type: 'USER_LOGIN',
+                    userLogin: this.state
+                })
+            }}>
+                <input placeholder="Insert Username" 
+                type="text"
+                value={this.state.username}
+                onChange={event => {
+                    this.setState({username: event.target.value})
+                }} />
+                <input placeholder="Insert Password" 
+                type="text"
```

Comment:
On line 32, could we possibly change the type attribute from `text` to `password`? This will make it so that the password is hidden with little dots. 😁